src/audio/buffer.c

```diff
@@ -38,7 +38,7 @@ struct comp_buffer *buffer_alloc(uint32_t size, uint32_t caps, uint32_t align)
 		return NULL;
 	}
 
-	buffer->addr = rballoc_align(RZONE_BUFFER, caps, size, align);
+	buffer->addr = rballoc_align(RZONE_BUFFER, caps, size, align, false);
 	if (!buffer->addr) {
 		rfree(buffer);
 		trace_buffer_error("buffer_alloc() error: "
```

Comment:
@mrajwa  But we will still see errors logged here right?

Reply:
@dbaluta, no you will see only regular log message which shall be treated as a worning. Previously it was error message and people were confused, linking almost every WoV issue with this error message like "Ohh of course it happens because there was not enough memory".